environments/getting-started.md

```diff
@@ -16,21 +16,34 @@ existing environments. Click the **New Environment** button.
 
 ![Create an Environment](../assets/create-env.png)
 
-Enter a friendly name for your environment, along with an
-[Image](../images/index.md) to use.
+1. Enter a friendly name for your environment, and choose an
+   [Image](../images/index.md) to use.
 
-When done, click **Create** to proceed. Coder redirects you to an overview page
-for your environment during the build process. Learn more about the Environment
-[creation parameters](./environment-params.md).
+1. If you want your environment to turn on at a specific time automatically,
+   toggle **Autostart** to **Yes**. You can set the autostart time in [User
+   Preferences](preferences.md#autostart). Please note that Coder disables
+   autostart if your environment is not off or hasn't been opened for three or more
+   days.
```

Comment:
Can I ask you to change:

"Please note that Coder disables autostart"

to

"Please note that autostart will not trigger" 

?